cmake/README.md

```diff
@@ -0,0 +1,47 @@
+# CMake system description
+
+The project uses CMake for integration and testing purposes.
+
+Conguration phase can be customized by passing additional variables: `cmake -D<var1>=<val1> -D<var2>=<val2> ... -D<varN>=<valN> <sources-dir>`
+
+The following variables are provided for oneDPL configuration:
+
+| Variable                       | Type   | Description                                                                                   | Default value |
+|--------------------------------|--------|-----------------------------------------------------------------------------------------------|---------------|
+| ONEDPL_USE_PARALLEL_POLICIES   | BOOL   | Enable parallel policies                                                                      | ON            |
+| ONEDPL_BACKEND                 | STRING | Threading backend; supported values: tbb, sycl, sycl_only, ...                                | tbb           |
+| ONEDPL_DEVICE_TYPE             | STRING | Device type, applicable only for sycl backends; supported values: GPU, CPU, FPGA_HW, FPGA_EMU | GPU           |
+| ONEDPL_USE_UNNAMED_LAMBDA      | BOOL   | Pass `-fsycl-unnamed-lambda` compile option                                                   | OFF           |
+| ONEDPL_USE_RANGES_API          | BOOL   | Enable the use of ranges API for algorithms                                                   | OFF           |
+| ONEDPL_FPGA_STATIC_REPORT      | BOOL   | Enable the static report generation for the FPGA_HW device type                               | OFF           |
+| ONEDPL_USE_OFFLINE_COMPILATION | BOOL   | Enable the offline compilation via OpenCL™ Offline Compiler (OCLOC)                           | OFF           |
+| ONEDPL_COMPILE_ARCH            | STRING | Architecture options for the offline compilation                                              | *             |
```

Comment:
Why we have `ONEDPL` prefix for all of variables if Cmake is not a part of oneAPI, but oneDPL only? Can we live with `BACKEND`, `DEVICE_TYPE`, ... without such prefix?

Reply:
This prefix is added to avoid possible conflicts with variables from client project that includes oneDPL sources through CMake, it's a kind of namespace. Currently I can't prove such conflicts, but they are possible in theory. If you have strong preference to get rid of the prefix I would not object, but it's a public interface, so it should be pretty solid.